README.md

```diff
@@ -34,3 +34,20 @@ In general, changes that require a PR (new functionality, new language) should i
 #### Deprecations
 
 We are moving away from using the `tx` cli, so the `.tx/config` file will eventually be deprecated.
+
+## Committing
+
+This project uses [semantic release](https://github.com/semantic-release/semantic-release) to ensure version bumps
+follow semver so that projects depending on it don't break unexpectedly.
+
+In order to automatically determine version updates, semantic release expects commit messages to follow the
+[conventional-changelog](https://github.com/bcoe/conventional-changelog-standard/blob/master/convention.md)
+specification.
+
+You can use the [commitizen CLI](https://github.com/commitizen/cz-cli) to make commits formatted in this way:
+
+```bash
+npm install -g commitizen@latest cz-conventional-changelog@latest
+```
+
+Now you're ready to make commits using `git cz`.
```

Comment:
I think it would be a good idea to include examples of what things are which kind of release. i.e.

- fix/chore: updating translations from transifex (changes z in x.y.z)
- feat: adding a new language (changes y in x.y.z), adding a new tool/script
- breaking: changing the structure of the exports (e.g. the supported langs json); changing the scripts used by other repos in incompatible way (changes x in x.y.z)

Reply:
Ah - that's a good idea, especially given the conversation about `fix` vs `chore` :)

Reply:
I added some examples and a link to more